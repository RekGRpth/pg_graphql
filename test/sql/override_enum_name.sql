begin;
    create type account_priority as enum ('high', 'standard');
    comment on type public.account_priority is E'@graphql({"name": "CustomerValue"})';
    select graphql.rebuild_schema();

    select name from graphql.type where enum = 'public.account_priority'::regtype;

rollback;
