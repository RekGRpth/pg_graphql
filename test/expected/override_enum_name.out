begin;
    create type account_priority as enum ('high', 'standard');
    comment on type public.account_priority is E'@graphql({"name": "CustomerValue"})';
    select name from graphql.type where enum = 'public.account_priority'::regtype;
     name      
---------------
 CustomerValue
(1 row)

rollback;
